packages/expo-updates/android/build.gradle

```
@@ -11,8 +11,8 @@ buildscript {
       // We can remove this path once we update the test environment
       // to use the same version of Kotlin as the `expo/expo` repo.
       def kotlinVersion = rootProject["kotlinVersion"]
-      if (kotlinVersion == "2.0.21") {
-        return "2.0.21-1.0.28"
+      if (kotlinVersion == "2.1.20") {
+        return "2.1.20-2.0.1"
```

```suggestion
      if (kotlinVersion == "2.1.20") {
        return "2.1.20-2.0.1"
      } else if (kotlinVersion == "2.0.21") {
        return "2.0.21-1.0.28"
      }
```

backward compatibility. or maybe use `when` expression